logic/transaction.js

```diff
@@ -47,7 +47,7 @@ const __private = {};
  * @todo Add description for the params
  */
 class Transaction {
-	constructor(db, ed, schema, genesisBlock, account, logger, cb) {
+	constructor(db, storage, ed, schema, genesisBlock, account, logger, cb) {
```

Comment:
Why the `db` is left out here? Is it still in use?

Reply:
Not in use. Removed and updated related modules.